web/src/lib/components/Navbar.svelte

```diff
@@ -46,13 +46,51 @@
         }
     }
 
+    function closeMenu() {
+        menuDisplay = "hidden";
+    }
+
+    function openMenu() {
+        menuDisplay = "block";
+    }
+
+    function toggleMenu() {
+        if ( menuDisplay === "block") {
+            closeMenu();
+        } else if ( menuDisplay === "hidden") {
+            openMenu();
+        }
+    }
+
     function toggleTheme() {
         let html = <HTMLHtmlElement>document.querySelector('html');
-        let toggle = <HTMLInputElement>document.getElementById('theme-toggle');
-        html.dataset.theme = toggle.checked ? 'night' : 'light';
+        console.log(html.dataset.theme );
+        if ( html.dataset.theme === 'night') {
```

Comment:
I assume this will break if we decide that the "dark" theme is another one (not "night"). It is better to check the state of the toggle rather than assuming we are using some particular theme.